grails-app/services/au/org/ala/biocollect/CommentService.groovy

```diff
@@ -16,17 +16,25 @@ class CommentService {
     }
 
     def deleteComment(data){
-        webService.deleteWrapper(grailsApplication.config.ecodata.service.url + "/comment/${data['id']}?userId=${data['userId']}")
+        Boolean alaAdmin = authService.userInRole(grailsApplication.config.security.cas.adminRole)
```

Comment:
I do not think we support FC_ADMIN. I have created a function userIsAlaAdmin in UserService.